### PR TITLE
Milestone 3: Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by grauJavier**
# New Features
- Section **Contact me** with HTML forms tags.
- **CSS Styling** applied to the contact form.

## Personal work
- Re-edition to **HTML id/class** format and **CSS cascade flow.**

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @grauJavier @GreetyCr , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not followed the work and submission mode requirement. below is the screenshot of the submission requirement. every student is expected to have a commit in this project but currently on @grauJavier has committed to this project, please kindly work on this and resubmit for a proper review :+1: 

inv

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,205 +1,274 @@
...
+        <h2 class="work-title-text">Contact me</h2>
+        <p class="body-text">If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form class="container" id="contact-me-form"
+          action="https://formspree.io/f/mnqywvqy"
+          method="POST">
+          <ul class="container">
+            <li>
+              <input class="contact-me-input" type="text" name="name" placeholder="Lorem Ipsum"/>
```
Comment (IngDavidTM) [inline index.html]: - [ ] Although you have successfully designed the input element for the name :+1:, please ensure that you add the "maxlength" and "required" attributes to the element as required by the project. The "maxlength" attribute will limit the length of the input to 30 characters while the "required" attribute will ensure that the user inputs their name before submitting the form.

File: index.html [hunk 2]
```diff
@@ -1,205 +1,274 @@
...
+            </li>
+            <li>
+              <input class="contact-me-input" type="email" name="email" placeholder="loremipsum@gmail.com"/>
+            </li>
+            <li>
+              <textarea class="contact-me-input" id="message-box" name="message" placeholder="Write your message here..."></textarea>
```
Comment (IngDavidTM) [inline index.html]: - [ ] Although you have successfully designed the text area :+1:, please ensure that you add the "maxlength" and "required" attributes to the element as required by the project. The "maxlength" attribute will limit the length of the input to 500 characters while the "required" attribute will ensure that the user inputs their name before submitting the form.

File: index.html [hunk 3]
```diff
@@ -1,205 +1,274 @@
...
+          I’d love to help with it!</p>
+        <form class="container" id="contact-me-form"
+          action="https://formspree.io/f/mnqywvqy"
+          method="POST">
+          <ul class="container">
+            <li>
+              <input class="contact-me-input" type="text" name="name" placeholder="Lorem Ipsum"/>
+            </li>
+            <li>
+              <input class="contact-me-input" type="email" name="email" placeholder="loremipsum@gmail.com"/>
```
Comment (IngDavidTM) [inline index.html]: - [ ] Although you have successfully designed the input element for the email :+1:, please ensure that you add the "required" attribute to the element as required by the project. The "required" attribute will ensure that the user inputs their name before submitting the form.

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi Team 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by grauJavier** [comment]
# Thanks for your review @IngDavidTM! 
I think everything should be in place now.

**Review by DuaneDave — APPROVED** [review]
Hi @team 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Form input fields are correctly validated

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**